src/main/java/hudson/plugins/jira/selector/p4/JobIssueSelector.java

```diff
@@ -0,0 +1,18 @@
+package hudson.plugins.jira.selector.p4;
+
+import java.util.Set;
+
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import hudson.plugins.jira.JiraSite;
+import hudson.plugins.jira.selector.AbstractIssueSelector;
+
+public class JobIssueSelector extends AbstractIssueSelector {
+
+    @Override
+    public Set<String> findIssueIds(Run<?, ?> run, JiraSite site, TaskListener listener) {
+        // TODO Auto-generated method stub
```

Comment:
No, definitely not. I just wanted to have your opinion on the direction. It got late last night so i had to stop. I will either implement p4 plugin or remove dependencies on it and the selector.